proxy/frontend/core/src/main/java/org/apache/shardingsphere/proxy/frontend/netty/CDCChannelInboundHandler.java

```diff
@@ -218,7 +218,8 @@ private void processStopStreamingRequest(final ChannelHandlerContext ctx, final
     private void processDropStreamingRequest(final ChannelHandlerContext ctx, final CDCRequest request, final CDCConnectionContext connectionContext) {
         DropStreamingRequestBody requestBody = request.getDropStreamingRequestBody();
         checkPrivileges(request.getRequestId(), connectionContext.getCurrentUser().getGrantee(), backendHandler.getDatabaseNameByJobId(requestBody.getStreamingId()));
-        backendHandler.dropStreaming(connectionContext.getJobId());
+        String streamingId = request.getDropStreamingRequestBody().getStreamingId();
+        backendHandler.dropStreaming(streamingId);
```

Comment:
1, `request.getDropStreamingRequestBody()` could be replace with `requestBody`

2, Could we remove `jobId` in connectionContext?

Reply:
2. `jobId` will used at channal inactive, eg. client closed without execute `stop streaming`